runtime/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java

```diff
@@ -234,7 +234,15 @@ private static Object handleReadDataCorruption(CorfuPayloadMsg<ReadResponse> msg
     }
 
 
-
+    /**
+     * Asynchronously write to the logging unit.
+     * @param payload   The log data to write to the logging unit.
+     * @return          A CompletableFuture which will complete with the WriteResult once the
+     *                  write completes.
+     */
+    public CompletableFuture<Boolean> write(ILogData payload) {
+        return router.sendMessageAndGetCompletable(CorfuMsgType.WRITE.payloadMsg(new WriteRequest(payload)));
```

Comment:
yes.

this is not blocking us currently, so we can go ahead and merge this PR, but definitely keep this enhancement under consideration for the future.